restapi/serializers.py

```
@@ -68,9 +68,14 @@ def update(self, instance, validated_data):
 
     def validate(self, attrs):
         # user = self.context['request'].user
-        user_ids = [user['user'].id for user in attrs['users']]
-        if len(set(user_ids)) != len(user_ids):
-            raise ValidationError('Single user appears multiple times')
+        try:
```

think about what you're doing here with this exception handling. 
This is the equivalent of swallowing an exception. Even if there is a validation error, you will log it and return the attrs to the caller, making them think that the data is valid when it's not.

Exception Handling is more than just logging the exception